coordinator/core/manifest.go

```diff
@@ -201,6 +203,47 @@ func (m Manifest) Check(ctx context.Context, zaplogger *zap.Logger) error {
 	return nil
 }
 
+// CheckUpdate checks if the manifest is consistent and only contains supported values.
+func (m Manifest) CheckUpdate(ctx context.Context, originalPackages map[string]quote.PackageProperties, alreadyUpdatedPackages map[string]quote.PackageProperties) error {
+	if len(m.Packages) <= 0 {
+		return errors.New("no packages defined")
+	}
+
+	// Check if manifest update contains values which we normally should not update
+	for packageName, singlePackage := range m.Packages {
+		// Check if the original manifest does even contain the package we want to update
+		if _, ok := originalPackages[packageName]; !ok {
+			return errors.New("update manifest specifies a package which the original manifest does not contain")
+		}
+
+		// Check if singlePackages contains illegal values to update
+		if singlePackage.Debug != false || singlePackage.UniqueID != "" || singlePackage.SignerID != "" || singlePackage.ProductID != nil {
+			return errors.New("update manifest contains unupdatable values")
+		}
+
+		// Check if singlePackages does actually contain a SecurityVersion value
+		if singlePackage.SecurityVersion == nil {
+			return errors.New("update manifest does not specifiy a SecurityVersion to update")
+		}
+
+		// Check based on the original manifest
+		if originalPackages[packageName].SecurityVersion != nil && *singlePackage.SecurityVersion < *originalPackages[packageName].SecurityVersion {
+			return errors.New("update manifest tries to downgrade SecurityVersion of the original manifest")
+		}
+
+		// Checks if we already have an update manifest set, if it does contain the package and if it does, if it actually holds a value for SecurityVersion (which should always be the case, but let's go safe here)
```

Comment:
Also make sure that the new update manifest contains all the packages of alreadyUpdatedPackages.